README.md

```diff
@@ -0,0 +1,164 @@
+      }
+      ```
+  - Authenticate a user: POST /api/auth/login   
+  
+    The Online Bookstore API uses JWT (JSON Web Token) for securing API endpoints. JWTs are issued to users upon successful authentication and must be included in the Authorization header of subsequent requests to protected endpoints.After a user successfully logs in, the server issues a JWT. This token must be included in the Authorization header of subsequent requests to protected endpoints.
+    - Example of request body:
+      ```
+      {
+      "email": "john.doe@example.com",
+      "password": "securePassword123",
+      }
+      ```
+    - Example of response body:
+      ```
+      {
+      "token": "eyJhbGciOiJJ9.eyJzdWIiOiIxMjM0yfQ.SflKssw5c"
+      }
+      ```
+- ##### Book
+  - Get books from catalog: GET /api/books
+  - Get book for id: GET /api/books/{id}
+  - Create a new book: POST /api/books
+  - Update a specific book: PUT /api/books/{id}
+  - Delete a specific book: DELETE /api/books/{id}
+- ##### Category
+  - Get all books' categories: GET /api/categories
+  - Get a category by id: GET /api/categories/{id}
+  - Get all books by current category: GET /api/categories/{id}/books
+  - Add a new category (only for role "ADMIN"): POST /api/categories
+  - Update a current category (only for role "ADMIN"):PUT /api/categories/{id}
+  - Delete a current category (only for role "ADMIN"): DELETE /api/categories/{id}
+- ##### Shopping Cart
+  - Get a user's shopping cart: GET /api/cart
+  - Add book to the shopping cart: POST /api/cart
+  - Update quantity of a book in the shopping cart: PUT /api/cart/cart-items/{cartItemId}
+  - Remove a book from the shopping cart: DELETE /api/cart/cart-items/{cartItemId}
+- ##### Orders
+  - Create an order: POST /api/orders
+  - Get history of orders: GET /api/orders
+  - Get all OrderItems for a specific order: GET /api/orders/{orderId}/items
+  - Get a specific OrderItem within an order: GET /api/orders/{orderId}/items/{itemId}
+  - Update order status: PATCH /api/orders/{id}
+- #####  Swagger Documentation
+  - the API documentation at http://localhost:8080/swagger-ui/index.html.
+  - You can explore all available endpoints, their descriptions, request parameters, and response schemas.
+
+### Postman Collection 
+- **[Click to use collection](https://www.postman.com/rent-masters/workspace/online-bookstore-api/collection/34492567-9cf5612a-c2d4-40b0-a8e4-8f2e06f2f996?action=share&creator=34492567)**
+
+### Live Preview 💻
+Check out the live demo of the Book Store application:
+
+- [Live Demo](https://www.loom.com/share/c6710f4f1ba14b339a09f87667c4046c?sid=fd664c29-e2e8-4f5b-92be-cddac9c62bfa)
+
+### 🌟 Getting Started
+##### Prerequisites
+- Java 17+
+- Maven 4+
+- MySQL 8+
+- Docker
+##### Installation
+To run the application locally, follow these steps:  
+
+1. Clone the Repository
+```
+git clone https://github.com/AnyaRatusnyak/project-book-store.git  
+cd project-book-store
+```
```

Comment:
`bash` or `sh` is usually used for terminal command code snippets
```suggestion
1. Clone the Repository
```bash
git clone https://github.com/AnyaRatusnyak/project-book-store.git  
cd project-book-store
```
```